### DISCUSSION TITLE
How to add an ExtractionChain?

### DISCUSSION THREAD

**Discussion by scorego** [langflow-ai/langflow#755] [Q&A]
I want to use extraction chain with langflow. Extraction chain is implemented on LLMChain, with customized `output_parser` and `llm_kwargs`, and I cannot find how to configure these parameters in langflow UI. Is there any plan to support  customizing them?

![image](https://github.com/logspace-ai/langflow/assets/10768254/c7260f95-44ee-4986-ba05-8671b09b82ff)

SEE https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/chains/openai_functions/extraction.py#L46
```
def create_extraction_chain(
    schema: dict, llm: BaseLanguageModel, verbose: bool = False
) -> Chain:

    function = _get_extraction_function(schema)
    prompt = ChatPromptTemplate.from_template(_EXTRACTION_TEMPLATE)
    output_parser = JsonKeyOutputFunctionsParser(key_name="info")
    llm_kwargs = get_llm_kwargs(function)
    chain = LLMChain(
        llm=llm,
        prompt=prompt,
        llm_kwargs=llm_kwargs,
        output_parser=output_parser,
        verbose=verbose,
    )
    return chain
```

PS, I think langflow can provide a way to customize all the class field of LLMChain.
![image](https://github.com/logspace-ai/langflow/assets/10768254/e09968b6-e908-4f49-9f25-0c83b49b5af5)

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
